src/components/form/dt-tags/dt-tags.js

```diff
@@ -126,6 +129,29 @@ export class DtTags extends DtMultiSelect {
     `;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    var empty = true;
+
+    if (value != undefined) {
```

Comment:
This doesn't handle null. If you just do `if (value) {`, that will make sure it's not null, undefined, or blank (any falsey value)